README.md

```diff
@@ -0,0 +1,352 @@
+# TiKV Go Client
+
+TiKV provides two types of APIs for developers: the Raw Key-Value API and the Transactional Key-Value API. This document uses two examples to guide you through how to use the two APIs in TiKV. The usage examples are based on multiple nodes for testing. You can also quickly try the two types of APIs on a single machine.
+
+It is **not recommended or supported** to use both the raw and transactional APIs on the same keyspace.
+
+## Raw Key-Value API
+
+To use the Raw Key-Value API in applications developed in the Go language, take the following steps:
+
+1. Install the necessary packages.
+
+    ```bash
+    export GO111MODULE=on
+    go mod init rawkv-demo
+    go get github.com/pingcap/tidb@master
```

Comment:
why use tidb instead of `tikv/client-go`?

Reply:
Sorry, I just copy from the tikv website client-go page, it's out-of-date, I think we should update that too.

Reply:
Actually, we have provided simple examples https://github.com/tikv/client-go/tree/master/examples. I think add some links should be enough.